app/controllers/grade_entry_forms_controller.rb

```diff
@@ -223,4 +223,16 @@ def upload
     end
     redirect_to action: 'grades', id: @grade_entry_form.id
   end
+
+  def grade_distribution_data
+    new_labels = ['0 - 5']
+
+    (1..19).each do |i|
```

Comment:
Use `map` instead of `each` here to return an array that you can add to `new_labels`.